src/main/java/racingcar/View/TextInterface.java

```diff
@@ -0,0 +1,39 @@
+package racingcar.View;
+
+import camp.nextstep.edu.missionutils.Console;
+
+public class TextInterface {
+    public String enterPlayer() {
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+        return Console.readLine();
+    }
```

Comment:
출력할 내용을 상수 선언하여 사용하게 되면 가독성이 더 좋아질 것 같습니다!
*저는 예시를 보여드리기 위해 클래스 내에 상수를 선언하였지만 Constants 클래스 내에 정리하셔서 사용하면 될 것 같습니다!
```suggestion
public class TextInterface {
    private static final String ASK_MULTIPLE_CAR_NAMES = "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)";
    public String enterPlayer() {
        System.out.println(ASK_MULTIPLE_CAR_NAMES);
        return Console.readLine();
    }
```